### ISSUE TITLE
Description of comment-descriptions option

### ISSUE THREAD

**Issue by glensc** [cjoudrey/graphql-schema-linter#324]
```
  --comment-descriptions

    use old way of defining descriptions in GraphQL SDL
```

can some more information be added what does this mean? what is the old and what is the new? 

and why this flag can't be put to config file?
